io_scene_niftools/modules/nif_export/animation/__init__.py

```diff
@@ -55,6 +55,9 @@ def set_flags_and_timing(self, kfc, exp_fcurves, start_frame=None, stop_frame=No
         # fill in the non-trivial values
         kfc.flags = 8  # active
         kfc.flags |= self.get_flags_from_fcurves(exp_fcurves)
+        if bpy.context.scene.niftools_scene.game == 'SID_MEIER_S_PIRATES':
+            # Sid Meier's Pirates! want the manager_controlled flag set
+            kfc.flags |= 32
```

Comment:
Instead of doing an OR, you could do kfc.flags.manager_controlled = True (although you'd also have to change `kfc.flags = 8` into `kfc.flags.value = 8`).

Reply:
If I change the above to

```
        kfc.flags_value = 8  # active
        kfc.flags |= self.get_flags_from_fcurves(exp_fcurves)
        if bpy.context.scene.niftools_scene.game == 'SID_MEIER_S_PIRATES':
            # Sid Meier's Pirates! want the manager_controlled flag set
            kfc.flags.manager_controlled = True
```

Would the `kfc.flags |= ...` line still be fine?

Reply:
Yes, that's still fine because the in-place or operator for bitfields was set up such that it adjusts the value.

However, don't forget the dot `.` in `flags._value`, otherwise you're not changing `flags` but adding a new attribute called `flags_value` to `kfc`.